### ISSUE TITLE
Get diff of commits

### ISSUE THREAD

**Issue by meyskens** [src-d/gitbase#553]
This is a feature request: it would be very handy to be able to query the files touched per commit. Maybe adding on that the ability to get either the old and the new blob or a diff of them.

**Comment by smola** [comment]
This would enable a whole new class of applications, but implementation needs to be designed carefully.

**Comment by campoy** [comment]
This is actually a very important feature for basically any demo where we analyze what an individual developer wrote.

What lines did they write? How many lines per language?

**Comment by erizocosmico** [comment]
Do we want this @ajnavarro?

**Comment by ajnavarro** [comment]
yes, but we didn't agree on how to do it. We should keep this open to keep it in mind.
